src/lp_sensitivity.jl

```diff
@@ -0,0 +1,340 @@
+                    con_basis_status = MOI.NONBASIC_AT_UPPER
+                elseif S <: MOI.GreaterThan && con_basis_status == MOI.NONBASIC
+                    con_basis_status = MOI.NONBASIC_AT_LOWER
+                end
+                push!(basis_status, con_basis_status)
+            end
+        end
+    end
+    return basis_status
+end
+
+function lp_rhs_perturbation_range(constraint::ConstraintRef{Model, _MOICON{F, S}}; feasibility_tolerance::Float64 = 1e-8
+                      )::Tuple{Float64, Float64} where {S <: Union{MOI.LessThan, MOI.GreaterThan, MOI.EqualTo}, T, F <: Union{MOI.ScalarAffineFunction{T}, MOI.SingleVariable}}
+    model = owner_model(constraint)
+    if termination_status(model) != MOI.OPTIMAL
+        error("The perturbation range of rhs is not available because the current solution "*
+              "is not optimal.")
+    end
+
+    try
+        con_status = MOI.get(model, MOI.ConstraintBasisStatus(), constraint)
+        # The constraint is inactive.
+        if con_status == MOI.BASIC
+            if S <: MOI.LessThan
+                return value(constraint) - constraint_object(constraint).set.upper, Inf
+            end
+            if S <: MOI.GreaterThan
+                return -Inf, value(constraint) - constraint_object(constraint).set.lower
+            end
+            return 0.0, 0.0
+        end
+    catch e
+        error("The perturbation range of rhs is not available because the solver doesn't "*
+              "support ´ConstraintBasisStatus´.")
+    end
+    # The constraint is active.
+    # TODO: This could be made more efficient because actually we only need the basic columns.
+    A, L, U, constraints = _std_matrix(model)
+    var_basis_status = _std_basis_status(model)
+    x = _std_primal_solution(model, constraints)
+    # Ax = 0 => B*x_B = -N x_N =: b_N
+    # Perturb row i by Δ => ̃x_B = B^-1 (b_N + Δ e_i) = x_B + Δ ρ
+    # Perturb var bound j by Δ => ̃x_B = B^-1 (b_N - Δ N_j) = x_B + Δ ρ
+    # Find min and max values of Δ : L_B <= ̃x_B <= U_B
+    # L_B <= x_B + Δ ρ <= U_B  -> L_B - x_B <= Δ ρ <= U_B - x_B
+    # Δ <= (U_B - x_B)_+ ./ ρ_+,  (L_B - x_B)_- ./ p_-
+    # Δ >= (U_B - x_B)_- ./ ρ_-,  (L_B - x_B)_+ ./ p_+
+    basic = var_basis_status .== Ref(MOI.BASIC)
+    B = A[:, basic]
+    x_B = x[basic]
+    local rho::Vector{Float64}
+    if F <: MOI.SingleVariable
+        var = constraint_object(constraint).func
+        j = findfirst(isequal(var), all_variables(model))
+        rho = -B \ Vector(A[:, j])
+    else
+        i = findfirst(isequal(constraint), constraints)
+        ei = zeros(size(A, 1))
+        ei[i] = 1.0
+        rho = B \ ei
+    end
+    UmX_B = U[basic] - x_B
+    LmX_B = L[basic] - x_B
+    pos = rho .> 0.0
+    neg = rho .< 0.0
+    # Find the first basic variable bound that is strictly violated.
+    # Use the vanilla 1e-8 tolerance (c.f. "Computational Techniques of the Simplex Method" by István Maros section 9.3.4.)
```

Comment:
Please move this tolerance up to where the `1e-8` appears, and mention that this tolerance should match the feasibility tolerance used by the LP solver.